.github/workflows/cflite_pr.yml

```diff
@@ -0,0 +1,34 @@
+name: ClusterFuzzLite PR fuzzing
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - '**'
+permissions: read-all
+jobs:
+  PR:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sanitizer: [address]
+    steps:
+    - name: Build Fuzzers (${{ matrix.sanitizer }})
+      id: build
+      uses: google/clusterfuzzlite/actions/build_fuzzers@v1
+      with:
+        sanitizer: ${{ matrix.sanitizer }}
+        language: python
```

Comment:
I think you needed to add this.
It's confusing that you need to specify language twice

Reply:
awesome, thanks! The doubling is in essence fine for my part but it should probably be documented here https://google.github.io/clusterfuzzlite//build-integration/python-lang/